README.md

```diff
@@ -35,38 +35,39 @@ Foreman is a mature project, deployed in [many organizations](http://projects.th
+![Hosts list](http://i.imgur.com/VMMLRd3.png)
+![New host](http://i.imgur.com/wl9MCyz.png)
+![EC2](http://imgur.com/x6gCogZ.png)
+![Provisioning templates](http://imgur.com/J3szFIu.png)
+![Subnets](http://imgur.com/isBcyGb.png)
+![Compute Resource](http://imgur.com/BetWNzW.png)
+![Edit Puppet class](http://imgur.com/0KDClmy.png)
+![Reports](http://imgur.com/NxlP7yo.png)
+![Statistics](http://imgur.com/DKGjtFQ.png)
+![Partition tables](http://imgur.com/Gg80lzg.png)
+![Installation media](http://imgur.com/BsYcvoM.png)
 
 # Installation
-Read the [quickstart section](http://theforeman.org/manuals/latest/#2.Quickstart) of the manual. If you know your setup has some specific needs, read the [installation scenarios section](http://theforeman.org/manuals/latest/#3.2.3InstallationScenarios).
+Read the [quickstart section](https://theforeman.org/manuals/latest/quickstart_guide.html#QuickstartGuide) of the manual. If you know your setup has some specific needs, read the [installation scenarios section](https://theforeman.org/manuals/latest/#3.2.3InstallationScenarios).
 
 # Documentation
 Our main documentation reference is the [Foreman manual](http://theforeman.org/manuals/latest/). If you find some gaps you would like to fill in the manual, please contribute in [this repo](https://github.com/theforeman/theforeman.org).
 
 ## API
-We document our API using [apipie](https://github.com/Apipie/apipie-rails).The [API chapter](http://theforeman.org/manuals/latest/index.html#5.1API) has more information about accessing the API and the layout of requests and responses. Also see the [reference documentation](http://theforeman.org/api_v2.html) available on our website, or via your own Foreman installation by appending `/apidoc` to the URL to see the API routes available.
+We document our API using [apipie](https://github.com/Apipie/apipie-rails).The [API chapter](http://theforeman.org/manuals/latest/index.html#5.1API) has more information about accessing the API and the layout of requests and responses. Also see the [reference documentation](https://theforeman.org/api/) available on our website, or via your own Foreman installation by appending `/apidoc` to the URL to see the API routes available.
 
 # Plugins
-Plugins are tools to extend and modify the functionality of Foreman. They are implemented as Rails engines that are packaged as gems and thus easily installed into Foreman. Read the [plugins section](http://theforeman.org/manuals/latest/index.html#6.Plugins) of the manual for more information.
+Plugins are tools to extend and modify the functionality of Foreman. They are implemented as Rails engines that are packaged as gems and thus easily installed into Foreman. Read the [plugins page](http://theforeman.org/plugins/) of our website for more information.
 
-An up-to-date plugin list is kept in the [wiki](http://projects.theforeman.org/projects/foreman/wiki/List_of_Plugins)
+An up-to-date plugin list is kept in [here](http://theforeman.org/plugins/)
```

Comment:
Can we avoid naming the link "here". How about "The [plugins page] lists all available plugins and has more information."